rs/tests/sdk/dfx_check_old_wallet_version_test.rs

```diff
@@ -58,8 +58,11 @@ fn test(env: TestEnv) {
 
     dfx.version();
 
+    let path = std::env::var("WALLET_CANISTER_0_7_2_WASM")
+        .expect("Environment variable 'WALLET_CANISTER_0_7_2_WASM' should be set");
+
     let wallet_wasm_path: PathBuf =
-        fs::canonicalize(get_dependency_path(WALLET_CANISTER_0_7_2_WASM)).unwrap();
```

Comment:
Can the `WALLET_CANISTER_0_7_2_WASM` const be removed as well?

Reply:
Woops thanks!